### PR TITLE
Actionbar

### PR DESCRIPTION
**Pull request by pelord**
**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines:https://github.com/infra-geo-ouverte/igo2/blob/master/.github/CONTRIBUTING.md#git-commit-guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What is the current behavior?** (You can also link to an open issue here)
Icons and tooltip  inside actionbar are static.

**What is the new behavior?**
Now, the icon or tooltip  inside actionbar could be modified and monitored by observable.

**Does this PR introduce a breaking change?** (check one with "x")
- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by pelord** [comment]
jIjVH6NRM1

**Comment by pelord** [comment]
iwJ2tROAV2

File: packages/common/src/lib/action/actionbar/actionbar-item.component.ts [hunk 1]
```diff
@@ -108,6 +106,24 @@ export class ActionbarItemComponent implements OnInit, OnDestroy {
         .subscribe((ngClass: {[key: string]: boolean}) => this.updateNgClass(ngClass));
     }
 
+    if (this.action.icon !== undefined) {
```
Comment (cbourget) [inline packages/common/src/lib/action/actionbar/actionbar-item.component.ts:109]: I don't think this check is needed is it?

File: packages/common/src/lib/action/actionbar/actionbar-item.component.ts [hunk 2]
```diff
@@ -108,6 +106,24 @@ export class ActionbarItemComponent implements OnInit, OnDestroy {
...
+      if (isObservable(this.action.icon)) {
+        this.icon$$ = this.action.icon
+          .subscribe((icon: string) => this.updateIcon(icon));
+      } else {
+        this.updateIcon(this.action.icon);
+      }
+    }
+
+    if (this.action.tooltip !== undefined) {
```
Comment (cbourget) [inline packages/common/src/lib/action/actionbar/actionbar-item.component.ts:118]: Same here

**Pull request merged**